From 53656ebfa3a8799c8641c2ff5493921f778da0a6 Mon Sep 17 00:00:00 2001
From: Jay Logue <jaylogue@google.com>
Date: Fri, 18 May 2018 11:14:30 -0700
Subject: [PATCH 3/9] openweave-esp32-lwip-patch-0003 : Removed include in
 lwipopts.h

Removed unnecessary inclusion of sys/ioctl.h in lwipopts.h that results in
compilation errors in certain contexts where LwIP headers are included.
---
 include/lwip/port/lwipopts.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/lwip/port/lwipopts.h b/include/lwip/port/lwipopts.h
index 70940f4..40df825 100644
--- a/include/lwip/port/lwipopts.h
+++ b/include/lwip/port/lwipopts.h
@@ -52,7 +52,6 @@
 #include <unistd.h>
 #include <sys/time.h>
 #include <sys/fcntl.h>
-#include <sys/ioctl.h>
 #include "esp_task.h"
 #include "esp_system.h"
 #include "sdkconfig.h"
-- 
1.9.1

